website/www/site/content/en/blog/beam-2.19.0.md

```diff
@@ -25,7 +24,7 @@ limitations under the License.
 -->
 
 We are happy to present the new 2.19.0 release of Beam. This release includes both improvements and new functionality.
-See the [download page]({{ site.baseurl }}/get-started/downloads/#2190-2020-02-04) for this release.<!--more-->
```

Comment:
Won't this change cause issues with staging?

Reply:
@robertwb: Hugo actually handles urls behind the scene (a great feature compared to Jekyll :D). You may check this out [1] for more details. In short, we only need to add `canonifyURLs = true` in `website/www/site/config.toml`.

[1] https://gohugo.io/content-management/urls/#canonicalization